c7n/cli.py

```diff
@@ -256,6 +261,7 @@ def main():
             command = getattr(
                 importlib.import_module(command.rsplit('.', 1)[0]),
                 command.rsplit('.', 1)[-1])
+        setproctitle(' '.join([sys.argv[0], options.subparser]))
```

Comment:
this doesn't look quite right, this just blanks the args, when really just want looking to shift them ' '.join(sys.argv[1:])

ie on a standard custodian run, this ends up pointing to `python`, and we want it to point to `custodian`, so we can disambiguate in watch dogs more easily from other python processes on the host.